pretty_cron/api.py

```diff
@@ -65,6 +70,12 @@ def _pretty_date(month_day, month, week_day):
 
 
 def _human_month(month):
+    if type(month)==tuple:
```

Comment:
`type(x) == tuple` is not the greatest, `isinstance(x, tuple)` is preferable. I don't think it makes such a difference in this case, but it's a good habit.